templates/helpers/toggleDetail.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = function (key) {
+  var toggleFor = ['SYSTEM OF ARRANGEMENT', 'COPYRIGHT', 'HISTORY NOTE'];
+
+  return toggleFor.indexOf(key.toUpperCase()) > -1;
+};
```

Comment:
You'll need to register this helper in https://github.com/TheScienceMuseum/collectionsonline/blob/master/client/templates.js for it to work after loading the page with javascript